### ISSUE TITLE
How can I ignore URL imports?

### ISSUE THREAD

**Issue by stefanfrede** [import-js/eslint-plugin-import#1988]
I have this import and would like to ignore it: `import confetti from 'https://cdn.skypack.dev/canvas-confetti';`.

I tried the following in my `.eslintrc.js`, but it did not work:
```
  settings: {
    'import/ignore': ['^(https://cdn.skypack.dev/)'],
  },
```

I also tried something simpler without success:
```
  settings: {
    'import/ignore': ['^https'],
  },
```

Now I'm wondering what I have to do to ignore imports like that.

**Comment by ljharb** [comment]
When you say "it did not work", can you elaborate?

**Comment by stefanfrede** [comment]
With it did not work I mean that my linter is still throwing the error `import/no-unresolved: Unable to resolve path to module 'https://cdn.skypack.dev/canvas-confetti'.`.

**Comment by ljharb** [comment]
See https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unresolved.md#ignore - you need to configure the no-unresolved rule separately.

**Comment by stefanfrede** [comment]
Thank you, but it seems I'm still doing something wrong.

I changed my settings now to the following:
```
  settings: {
    'import/no-unresolved': [2, { ignore: ['^https'] }],
  },
```
Sadly this does not change anything.

**Comment by ljharb** [comment]
what's your full eslint config?

**Comment by stefanfrede** [comment]
This is my config:
```
module.exports = {
  root: true,
  extends: [
    'eslint:recommended',
    'plugin:import/errors',
    'plugin:import/warnings',
    'plugin:prettier/recommended',
  ],
  parserOptions: {
    ecmaVersion: 11,
    sourceType: 'module',
    allowImportExportEverywhere: true,
    ecmaFeatures: {
      impliedStrict: true,
    },
  },
  env: {
    browser: true,
    node: true,
    es6: true,
  },
  plugins: ['html'],
  settings: {
    'import/no-unresolved': [2, { ignore: ['^https'] }],
  },
};
```

**Comment by ljharb** [comment]
(i'm confused by `allowImportExportEverywhere`, since import/export statements don't work except at Module level)

that definitely seems like it should work. You're not using any custom resolvers, you have root true, and none of those extended configs do anything wonky.

can you share the file that's being linted?

**Comment by stefanfrede** [comment]
Sure:
```
import Template from './template.js';

import confetti from 'https://cdn.skypack.dev/canvas-confetti';

export default class App extends HTMLElement {
  constructor() {
    super();

    this.attachShadow({ mode: 'open' });
    this.shadowRoot.innerHTML = Template.render();
    this.dom = Template.mapDOM(this.shadowRoot);

    confetti();
  }
}

if (!customElements.get('mf-app')) {
  customElements.define('mf-app', App);
}
```

**Comment by stefanfrede** [comment]
I pushed my code to GitHub: https://github.com/stefanfrede/monkey-funnel.

I removed the skypack import for now but maybe the rest of my code is helping to figure out  why `import/no-unresolved': [2, { ignore: ['^https'] }]` is not ignoring imports like `import confetti from 'https://cdn.skypack.dev/canvas-confetti'`.

**Comment by ljharb** [comment]
@stefanfrede `npm run lint:eslint` errors out on "no files matching src/".

If i run `npx eslint .` it does not produce an error.

**Comment by stefanfrede** [comment]
Yes, sorry, my bad. I adjusted the code and if you now run `npm run lint:eslint` you get the `import/no-unresolved` error.

This is the [.eslintrc.js](https://github.com/stefanfrede/monkey-funnel/blob/main/.eslintrc.js#L24) and this is the [file](https://github.com/stefanfrede/monkey-funnel/blob/main/components/foobar/foobar.js#L3) where the error is happening.

**Comment by soryy708** [comment]
The files linked for reproduction are no longer valid, it seems like perhaps the repository was deleted?

Cannot reproduce.
